Criteo doc updates (#808)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/criteo.mdx

description: Step-by-step guide on sending your event data from RudderStack to Criteo.
---

[Criteo](https://www.criteo.com/) is a popular online display advertising platform that offers various solutions to increase your website traffic, generate brand awareness, and boost sales.

RudderStack supports Criteo as a destination to which you can send your event data directly.

## Getting started

> Before configuring Criteo as a destination in RudderStack, verify if Criteo supports the source platform you are sending the events from. Refer to the following table:

Reviewer: Before configuring Criteo as a destination in RudderStack, verify if the source platform is supported by Criteo by referring to the table below:

> Before configuring Criteo as a destination in RudderStack, verify if Criteo supports the source platform you are sending the events from. Refer to the following table:

Author: ```suggestion
Before configuring Criteo as a destination in RudderStack, verify if the source platform is supported by Criteo by referring to the table below:
```

Once you have confirmed that the source platform supports sending events to Criteo, follow these steps:

1. From your [RudderStack dashboard](https://app.rudderstack.com/), add the source. Then, from the list of destinations, select **Criteo**.
2. Assign a name to the destination and click **Continue**.

## Connection settings

To successfully configure Criteo as a destination, you need to configure the following settings:

<img src="../../assets/event-stream-destinations/criteo-connection-settings-1.png" alt="Criteo connection settings" />
<img src="../../assets/event-stream-destinations/criteo-connection-settings-2.png" alt="Criteo connection settings" />

> <img src="../../assets/event-stream-destinations/criteo-connection-settings-3.png" alt="Criteo connection settings" />

Author: As discussed, the images need to be update to reflect the new setting.

Author: I've updated the images.

</div>

- **Email Hashing Method**: Criteo lets you use the email addresses in both hashed and non-hashed formats. If you choose **MD5**, RudderStack will hash-encode the email address before sending it to Criteo.
- **Map Specific Fields to Criteo Fields:** Enter the payload fields and the corresponding mapped fields that will be used to send the event data to Criteo.

<div class="infoBlock">
Criteo allows you to send any extra data about a page or a user to add more context to the events. You can set this feature in Criteo with the assistance of your Criteo Account Manager. You can then use the <strong>Map Specific Fields to Criteo Fields</strong> field mapping feature to send extra data through RudderStack.
</div>

- **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to Mouseflow.

<div class="infoBlock">
For more information on this setting, refer to the <Link to="/sources/sdks/event-filtering/">Client-side Events Filtering</Link> guide.
</div>

> - **Mapping RudderStack events to the Criteo standard events**: Enter the **Event Name** and the corresponding [**Standard Event**]() from the dropdown. This standard event will be triggered when the mapped RudderStack event is called.

Author: @mihir-4116 please share the Criteo standard events reference link to be added here. Is [this](https://www.criteo.com/wp-content/uploads/2018/09/CSPOneTag_v1.1.pdf) a valid reference?

Reviewer 2: Yes @ameypv-rudder this looks good.

Reviewer 2: We can use this link.

> - **Mapping RudderStack events to the Criteo standard events**: Enter the **Event Name** and the corresponding [**Standard Event**]() from the dropdown. This standard event will be triggered when the mapped RudderStack event is called.

Author: ```suggestion
- **Mapping RudderStack events to the Criteo standard events**: Enter the **Event Name** and the corresponding [**Standard Event**](https://www.criteo.com/wp-content/uploads/2018/09/CSPOneTag_v1.1.pdf) from the dropdown. This standard event will be triggered when the mapped RudderStack event is called.
```

})
```

In the above example, RudderStack captures the information related to the `Product Viewed` event and the associated details such as quantity, price, category, etc.

> The following table details the mapping of the <Link to="/event-spec/ecommerce-events-spec/">RudderStack e-commerce events</Link> and the [Criteo OneTag events](https://support.criteo.com/s/article?article=All-Criteo-OneTag-events-and-parameters&language=en_US).

Reviewer: This criteo link is redirected to its home page. Maybe this should be the one: https://help.criteo.com/kb/guide/en/all-criteo-onetag-events-and-parameters-vZbzbEeY86/Steps/775825

> The following table details the mapping of the <Link to="/event-spec/ecommerce-events-spec/">RudderStack e-commerce events</Link> and the [Criteo OneTag events](https://support.criteo.com/s/article?article=All-Criteo-OneTag-events-and-parameters&language=en_US).

Author: ```suggestion
The following table details the mapping of the <Link to="/event-spec/ecommerce-events-spec/">RudderStack e-commerce events</Link> and the [Criteo OneTag events](https://help.criteo.com/kb/guide/en/all-criteo-onetag-events-and-parameters-vZbzbEeY86/Steps/775825).
```
